CHANGELOG.md

```diff
@@ -11,9 +11,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The Open Images format now supports bounding box annotations
   (<https://github.com/openvinotoolkit/datumaro/pull/352>).
 - Bounding boxes values decrement transform (<https://github.com/openvinotoolkit/datumaro/pull/366>)
+- Improved error reporting in Dataset (<https://github.com/openvinotoolkit/datumaro/pull/386>)
+
 ### Changed
 - Datumaro no longer depends on scikit-image
   (<https://github.com/openvinotoolkit/datumaro/pull/379>)
+- `Dataset` remembers export options on saving / exporting for the first time (<https://github.com/openvinotoolkit/datumaro/pull/386>)
```

Comment:
To me this seems like very surprising behavior. Saving seems like an operation that should not modify the internal state of the object.

Maybe instead of implicitly remembering options, you could add support for explicitly setting the default saving options for a dataset?

Reply:
An option to specify option explicitly already exists - it is the `bind()` method.

To me, binding of unbound dataset on the first save looks pretty natural, because the following saves use just the same parameters.

Reply:
Okay, I thought this over, and I realized that you can use `export` if you need to have control over the options that are used, and therefore `save` remembering the options should not be a problem.